xarray/core/groupby.py

```diff
@@ -98,14 +98,28 @@ def __init__(self, obj, group, squeeze=False, grouper=None):
             if the dimension is squeezed out.
         grouper : pd.Grouper, optional
             Used for grouping values along the `group` array.
+        group_bins : array-like, optional
```

Comment:
Yes, I agree. I just did this because `bins` was already used as a local variable elsewhere in `__init__`. But I can change this for sure.